GitHubReactions.ps1

```diff
@@ -0,0 +1,363 @@
+    $RepositoryName = $elements.repositoryName
+
+    $telemetryProperties = @{
+        'OwnerName' = (Get-PiiSafeString -PlainText $OwnerName)
+        'RepositoryName' = (Get-PiiSafeString -PlainText $RepositoryName)
+    }
+
+    $uriFragment = "/repos/$OwnerName/$RepositoryName/issues/$Issue/reactions"
+    $description = "Setting reaction $ReactionType for Issue $Issue in $RepositoryName"
+
+    $params = @{
+        'UriFragment' = $uriFragment
+        'Description' =  $description
+        'Method' = 'Post'
+        'Body' = @{ content = $ReactionType } | ConvertTo-Json
+        'AcceptHeader' = 'application/vnd.github.squirrel-girl-preview+json'
+        'AccessToken' = $AccessToken
+        'TelemetryEventName' = $MyInvocation.MyCommand.Name
+        'TelemetryProperties' = $telemetryProperties
+        'NoStatus' = (Resolve-ParameterWithDefaultConfigurationValue -Name NoStatus -ConfigValueName DefaultNoStatus)
+    }
+
+    $result = Invoke-GHRestMethod @params
+    if ($PSBoundParameters.ContainsKey('PassThru')) {
+        return $result
+    }
+}
+
+function Remove-GitHubReaction
+{
+<#
+    .SYNOPSIS
+        Removes a reation on a given GitHub issue.
+
+    .DESCRIPTION
+        Removes a reation on a given GitHub issue.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER OwnerName
+        Owner of the repository.
+        If not supplied here, the DefaultOwnerName configuration property value will be used.
+
+    .PARAMETER RepositoryName
+        Name of the repository.
+        If not supplied here, the DefaultRepositoryName configuration property value will be used.
+
+    .PARAMETER Uri
+        Uri for the repository.
+        The OwnerName and RepositoryName will be extracted from here instead of needing to provide
+        them individually.
+
+    .PARAMETER Issue
+        The issue number.
+
+    .PARAMETER ReactionId
+        The Id of the reaction. You can get this from using Get-GitHubReaction.
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .PARAMETER NoStatus
+        If this switch is specified, long-running commands will run on the main thread
+        with no commandline status update.  When not specified, those commands run in
+        the background, enabling the command prompt to provide status information.
+        If not supplied here, the DefaultNoStatus configuration property value will be used.
+
+    .EXAMPLE
+        Remove-GitHubReaction -OwnerName PowerShell -RepositoryName PowerShell -Issue 12626 -ReactionId 1234
+
+        Remove a reaction by Id on Issue 12626 from the PowerShell\PowerShell project.
+
+    .EXAMPLE
+        Get-GitHubReaction -OwnerName PowerShell -RepositoryName PowerShell -Issue 12626 -ReactionType rocket | Remove-GitHubReaction
```

Comment:
Pretty awesome that you added support for this pipelining.  Getting that working throughout the whole module is definitely a priority (see #63).

Reply:
Commented on that